ui/src/assets/docs/de/faq-page.md

```diff
@@ -0,0 +1,136 @@
+
+In diesem Beispiel würden Sie folgende Fehlermeldung erhalten:
+
+TODO
+
+Lösung: Erstellen Sie mit Zustimmung des PSU einen neuen Consent mit Verwendung der gewünschten Endpoints.
+
+Ein anderer Grund könnte sein, dass ihr Consent ungültig ist. Der Consent Status wird automatisch auf "expired" gesetzt und damit ungültig sein, wenn das Datum im Feld "expirationDate" erreicht wurde.
+
+## &nbsp;
+
+## Nutzerauthentifizierung (SCA)
+
+**Wie kann ich den Transaktions- oder Consentstatus verändern?**\
+Eine neu angelegte Transaktion oder ein neu angelegter Consent hat im ersten Schritt den Status "received".
+Um die Authentifizierung (SCA) mittels Redirect abzudecken, stellt die Sandbox einen Redirect Server bereit, wo der Status sich je nach definiertem PSU verändert.
+Da der Redirect Ansatz für die Sandbox vereinfacht umgesetzt ist, reicht es aus mit einer PSU-ID mittels Query-Parameter die SCA zu simulieren. Folgendes Beispiel stellt die Umsetzung dar:
+
+`GET https://sandbox-api.dev.adorsys.de/v1/online-banking/init/pis/5239763c-8641-41cc-a354-d52d35da0c9e?psu-id=PSU-Successful`
+
+Der Query Parameter ist zwingend notwendig für die Durchführung. Wenn Sie nicht angefügt wird, zeigt der Redirect Server eine Standard-Fehlermeldung an.
+
+## &nbsp;
+
+**Warum kann ich den Transaktions- oder Consentstatus nicht von einem bestimmten PSU verändern**\
+Wenn der Transaktions- oder Consentstatus "received" lautet, kann es möglich sein, dass die PSU-ID nicht zur angegebenen IBAN in der Zahlungs- oder Consentanlage passt. Die definierten PSUs mit zugehörigen Testdaten finden Sie im develeoper-Portal dargestellt. <br />
+Wenn der Status sich trotz durchgefürter SCA und passendem PSU mit zugehöriger IBAN nicht verändert, achten Sie auf Typographie und Groß- und Kleinschreibung.
+
+## &nbsp;
+
+## Zertifikate
+
+**Wie kann ich ein valides Zertifikat generieren?**\
+Um auf die XS2A API zugreifen zu können, braucht ein Drittanbieter ein gültiges QWAC Zertifikat, welches zukünftig von registrierten Treuhanddienstleister (TSP=Trust Service Provider) ausgestellt wird. Um im adorsys PSD2 Accelerator Zugriff zu erhalten, können Sie sich hier ein Testzertifikat generieren:
+[https://sandbox-portal.dev.adorsys.de/app/certificate-service](https://sandbox-portal.dev.adorsys.de/app/certificate-service).
+Sie bekommen ein unterschriebenes Zertifikat (.pem) und den zugehörigen private
+key (.key) integriert in einer .zip file.
+
+## &nbsp;
+
+**Wie kann ich ein Zertifikat in einen Request integrieren?**\
+Wenn Sie ein QWAC Zertikat erhalten haben, müssen Sie es in in Ihren Request integrieren. Es gibt viele Tools, mit denen es möglich ist REST Calls durchzuführen. Wir erklären nachfolgend die Durchführung mit zwei weit verbreiteten Tools - cURL and Postman.
+cURL: Fügen Sie das .pem file mit `--cert ./certificate.p` und den private
+key mit `--key ./private.key` ein. Achten Sie darauf, dass Sie gegebenfalls den Pfand zu Ihren Dateien anpassen.
```

Comment:
Wenn Sie ein QWAC Zertifikat erhalten haben, müssen Sie es in Ihrem Request integrieren. Es gibt viele Tools mit REST Calls ausgeführt werden können. Wir erklären nachfolgend die Durchführung mit zwei weit verbreiteten Tools - cURL und Postman.
cURL: Fügen Sie die .pem Datei mit `--cert ./certificate.p` und den private key mit `--key ./private.key` in Ihren Request ein. Bitte beachten Sie, dass der relative Pfad zu Ihren Dateien gegebenenfalls angepasst werden muss.

Reply:
done